integreat_cms/cms/views/dashboard/dashboard_view.py

```diff
@@ -64,18 +64,20 @@ def get_unreviewed_pages_context(self):
         :return: Dictionary containing the context for unreviewed pages
         :rtype: dict
         """
-        unreviewed_pages = (
+        last_versions = (
```

Comment:
This query is now essentially repeated 4 times in the same view, right?
Maybe it would make sense to outsource the query for the latest versions (of all languages) into a separate function, set a class attribute with these ids and then use them in all the subsequent helper functions? We'd just have to move the respective language filters into the follow-up querysets.

Reply:
Thank you for review 👍 The suggestions are applied.

Reply:
Great! Yes, that's how I imagined it :+1: